sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -247,7 +247,7 @@ export namespace ClientEntityContext {
             connectionId,
             streamingReceiver.name
           );
-          await streamingReceiver.onDetached(error);
+          await streamingReceiver.onDetached(error, true);
```

Comment:
You absolutely do not need to change this _but_ I've recently been converting over from using booleans to using string unions instead.

so 

```
await streamingReceiver.onDetached(error, "fromConnection"); 
```

Which reads a bit nicer and is still type checked.

Helps a little bit when you're like "so...what is true here?"

Reply:
I changed the variable name but kept using a boolean (now stored in a descriptive variable).